doc/scripting_tr2_tr3.asc

```diff
@@ -129,18 +130,18 @@ Script Flags
 ----
 enum ScriptFlags
 {
-    DemoVersion =              (1 << 0), //(=1) Is this a demo version of the game
-    TitleDisabled =            (1 << 1), //(=1) Disables Title Screen
-    CheatModeCheckDisabled =   (1 << 2), //(=1) Related to in-game cheats?
-    NoInputTimeout =           (1 << 3), //(=1) Disables input timeout for demo mode
-    LoadSaveDisabled =         (1 << 4), //(=1) Disables loading/saving
-    ScreenSizingDisabled =     (1 << 5), //(=1) Disables screen resizing PC only?
-    LockoutOptionRing =        (1 << 6), //(=1) Stops options ring from being selected
-    DozyCheatEnabled =         (1 << 7), //(=1) Enable DOZY (sometimes they removed the internal code)
-    UseXor =                   (1 << 8), //(=1) If true all string chars (except null termination) must be xor-ed by XorKey.
-    GymEnabled =               (1 << 9), //(=1) Is Gym available on title screen.
-    SelectAnyLevel =           (1 << 10),//(=1) Unlocks all levels
-    EnableCheatCode =          (1 << 11) //(=1) Related to in-game cheats?
+    DemoVersion =              (1 << 0),  // If set, it indicates that the game is a demo distribution.
+    TitleDisabled =            (1 << 1),  // If set, it indicates that the game has no Title Screen.
+    CheatModeCheckDisabled =   (1 << 2),  // If set, it indicates that the game does not look for the cheat sequence keystrokes and events. 
+    NoInputTimeout =           (1 << 3),  // If set, it indicates that the game waits forever if there is no input (won't enter demo mode).
+    LoadSaveDisabled =         (1 << 4),  // If set, it indicates that the game does not allow save games.
+    ScreenSizingDisabled =     (1 << 5),  // If set, it indicates that the game does not allow screen resizing (with the function keys).
+    LockoutOptionRing =        (1 << 6),  // If set, it indicates that the user has no access to the Option Ring while playing the game. 
+    DozyCheatEnabled =         (1 << 7),  // If set, it indicates that the game has the DOZY cheat enabled (not present in final versions). 
+    UseXor =                   (1 << 8),  // If set, it indicates that a cypher byte was used to encrypt the strings in the script file, and is stored in the XorKey field. 
+    GymEnabled =               (1 << 9),  // Is Gym available on title screen.
+    SelectAnyLevel =           (1 << 10), // If set, it indicates that the names of the levels are all listed in the Passport. 
```

Comment:
Ambiguous, this just enables level select when New Game is selected at the title screen in the passport.